go.sum

```diff
@@ -1,4 +1,6 @@
+bitbucket.org/creachadair/shell v0.0.6 h1:reJflDbKqnlnqb4Oo2pQ1/BqmY/eCWcNGHrIUO8qIzc=
```

Comment:
This file shouldn't see changes without an accompanying change to go.mod. Fix with `go mod tidy`.